Gemfile

```diff
@@ -1,4 +1,9 @@
 source 'https://rubygems.org'
 gemspec :name => 'gollum-lib'
 gem 'irb'
-gem 'gollum-rugged_adapter', :git => 'https://github.com/gollum/rugged_adapter/', :branch => 'find_branches'
+
+if RUBY_PLATFORM == 'java'
+  gem 'gollum-rjgit_adapter', git: 'https://github.com/dometto/gollum-lib_rjgit_adapter/', branch: 'find_branch'
+else
+  gem 'gollum-rugged_adapter', git: 'https://github.com/gollum/rugged_adapter/', branch: 'find_branches'
+end
```

Comment:
Similar comment re: inline comment here. Having the context readily available about the conditions that require this temporary change would be nice, in my opinion.